src/osd/PeeringState.cc

```diff
@@ -6596,6 +6727,12 @@ boost::statechart::result PeeringState::GetLog::react(const QueryState& q)
   return forward_event();
 }
 
+boost::statechart::result PeeringState::GetLog::react(const QueryUnfound& q)
+{
+  // XXX: Should we report "state"; "GetLog", "have might_have_unfound": false
+  return discard_event();
```

Comment:
Oops should match QueryState with forward_event()

Reply:
Using discard_event() in all cases now that I understand what forward_event() is for.